test/tests/plone-addons/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -eo pipefail
+
+image="$1"
+
+cname="plone-container-$RANDOM-$RANDOM"
+cid="$(docker run -d -e PLONE_DEVELOP=src/eea.facetednavigation -e PLONE_ADDONS=eea.facetednavigation -e PLONE_ZCML=eea.facetednavigation-meta --name "$cname" "$image" cat)"
```

Comment:
This container will likely be dead by the time the`docker exec` below tries to execute and would then fail.  Maybe just replace it all with a `docker run -i --rm -e .... cat custom.cfg`.  Plus we would get to drop the `trap` :smile:.